packages/seller-client/src/components/OrderListItem/OrderListItem.tsx

```diff
@@ -0,0 +1,75 @@
+import type { ProductOptionGroup } from 'generated/graphql';
+
+import React from 'react';
+import { Box, Typography, ButtonBase } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import Price from 'components/Price';
+import Spacer from 'components/Spacer';
+
+const IMAGE_SIZE = 72;
+
+const useStyles = makeStyles(({ spacing }) => ({
+  container: {
+    width: '100%',
+    marginTop: spacing(0.25),
+    marginBottom: spacing(0.25),
+    display: 'flex',
+  },
+}));
+
+type OrderListItemProps = {
+  name: string;
+  price: number;
+  image?: { thumbnailSrcUrl: string; name: string };
+  groups?: ProductOptionGroup[];
+  onClick?: () => void;
+};
+
+const OrderListItem: React.FC<OrderListItemProps> = ({
+  image,
+  name,
+  price,
+  groups,
+  onClick,
+}) => {
+  const classes = useStyles();
+  return (
+    <ButtonBase className={classes.container} onClick={onClick}>
+      {image && (
+        <Box marginRight={1}>
+          <img
+            src={image.thumbnailSrcUrl}
+            alt={image.name}
+            width={IMAGE_SIZE}
+            height={IMAGE_SIZE}
+          />
+        </Box>
+      )}
+      <Box flex={1} paddingY={1}>
+        <Box display="flex" justifyContent="space-between">
+          <Typography variant="subtitle1">{name}</Typography>
+          <Typography variant="subtitle2">
+            <Price price={price} />
+          </Typography>
+        </Box>
+        {groups && (
+          <Box display="flex" flexWrap="wrap">
+            {groups.map((group) => (
+              <>
+                <Box key={group.name} display="flex" flex={1}>
+                  <Typography variant="body2" noWrap>
+                    {group.name}: {group.option}
+                  </Typography>
+                </Box>
```

Comment:
this `Box` only has one child so I don't think we need `display="flex"`
```suggestion
                <Box key={group.name} flex={1}>
                  <Typography variant="body2" noWrap>
                    {group.name}: {group.option}
                  </Typography>
                </Box>
```